docs/types/operators.rst

```diff
@@ -26,6 +26,24 @@ except for comparison operators where the result is always ``bool``.
 The operators ``**`` (exponentiation), ``<<``  and ``>>`` use the type of the
 left operand for the operation and the result.
 
+Ternary Operator
+----------------
+The ternary operator evaluates a given expression to a true or false value and then
+executes one of the two given statements depending upon the result of the evaluation.
```

Comment:
Technically, what you call ``statementA`` and ``statementB`` aren't statements, but also expressions...